prody/utilities/catchall.py

```diff
@@ -29,9 +29,9 @@ def calcModeClusters(similarities, n_clusters=0, linkage='all', method='tsne', c
     :type n_clusters: int
 
     :arg linkage: if **all**, will test all linkage types (ward, average, complete,
-                    single). Otherwise will use only the one given as input. Default is
+                    single). Otherwise will use only the one(s) given as input. Default is
                     **all**.
-    :type linkage: str
+    :type linkage: str, list
```

Comment:
I'd suggest adding numpy ndarray and tuple here if you use isListLike like I suggested.